HaxeManual/10-std.tex

```diff
@@ -868,12 +875,37 @@ \subsection{Implementation details}
 \section{Input/Output}
 \label{std-input-output}
 
-\section{Sys/sys}
+\section{Sys}
 \label{std-sys}
 
+The majority of Haxe targets are so-called ``sys'' targets. This means the targets have access to system APIs such as the filesystem, networking, threads, and more. The only non-sys targets supported by Haxe are Flash, JavaScript, and ActionScript 3, although JavaScript can support sys APIs when running under Node.js runtime.
+
+\paragraph{Related content}
+\begin{itemize}
+	\item See the \href{https://api.haxe.org/sys/}{sys package} on the API documentation for more details on its classes.
+\end{itemize}
+
+\subsection{Threading}
+\label{std-threading}
+
+\since{4.0.0}
+
+A \href{https://api.haxe.org/sys/thread/Thread.html}{unified threading API} is available on some sys targets. The compile-time define \expr{target.threaded} is set when the API is available. The API allows very simple creation of threads from functions:
+
+\haxe{assets/Threads.hx}
+
+All spawned threads are currently treated as daemon threads, meaning that the main thread will not wait for their completion.
+
+Due to the nature of threads accessing a shared memory space with all the Haxe variables and objects, it is possible to run into issues due to deadlocks and race conditions. The standard library provides some core synchronization constructs in the \href{https://api.haxe.org/sys/thread/}{\expr{sys.thread}} package.
```

Comment:
Phrasing is a bit weird, sounds like the nature of threads is doing something.

Reply:
Read it again and nevermind, I'm a bit weird.

Reply:
Apparently I'm weird too then, I also found that a bit hard to parse.

Reply:
Yeah that's a mistake, I accidentally a word.

> Due to threads having access to a shared memory space with all the Haxe variables and objects, it is possible ...

Is that better?

Reply:
Much better.